ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLog.java

```diff
@@ -405,12 +405,13 @@ protected CompletableFuture<Long> purgeImpl(long index) {
     try (AutoCloseableLock writeLock = writeLock()) {
       SegmentedRaftLogCache.TruncationSegments ts = cache.purge(index);
       updateSnapshotIndexFromStateMachine();
-      LOG.debug("purging segments:{}", ts);
       if (ts != null) {
+        LOG.info("{}: {}", getName(), ts);
         Task task = fileLogWorker.purge(ts);
         return task.getFuture();
       }
     }
+    LOG.info("{}: purge({}) found nothing to purge.", getName(), index);
```

Comment:
It seems this line will cause a lot of info log? If we don't actually delete the file here, maybe we don't need to log here

Reply:
Sure, let's change it to debug.